### DISCUSSION TITLE
FM RRW3_GET_QUERY_VIEW_DATA

### DISCUSSION THREAD

**Discussion by Eversince001** [SAP/PyRFC#354] [Q&A] [answered]
Hello everyone, I encountered a problem when calling FM RRW3_GET_QUERY_VIEW_DATA.
The query/view/provider and call parameters are transmitted to the FM input.
In my case I use query, the parameters for the call are as follows:
VAR_NAME1=ZHR_DATE2
VAR_VALUE_EXT_1=01.11.2023
VAR_NAME2=ZHR_DATE1
VAR_VALUE_EXT_2=11.10.2023
VAR_NAME3=ZHR_ORGUNIT_HIER_ALL
VAR_VALUE_EXT_3=1
![0873-d8dc-52f2-5a90](https://github.com/SAP/PyRFC/assets/57628681/6bf55596-8558-48ca-818a-b58578b9cf7e)

So, how can I pass a list of parameters to a given FM using Python (PyRFC)? none of my attempts were successful, Thanks!

**Comment by bsrdjan** [comment, marked as answer]
the same question answered here https://github.com/SAP/PyRFC/issues/348

**Comment by Eversince001** [comment]
Thanks!
